modules/RTC/ortc/RTCSessionDescription.js

```diff
@@ -6,20 +6,20 @@ import sdpTransform from 'sdp-transform';
 export default class RTCSessionDescription {
```

Comment:
How about adding some `toJSON` method to this class so it behaves like the one in Chrome / Firefox?